Examples/Modules/nuclear_fusion/analysis_proton_boron_fusion.py

```diff
@@ -0,0 +1,700 @@
+#! /usr/bin/env python
+# Copyright 2021 Neil Zaim
+#
+# This file is part of WarpX.
+#
+# License: BSD-3-Clause-LBNL
+
+import sys
```

Comment:
The the general file name treatment below:
```suggestion
import os
import sys
```